views/list-job-roles.html

```diff
@@ -8,23 +8,23 @@ <h2 style="color:red;font-weight:bold;">{{ errormessage }}</h2>
 {% endif %}
 
 {% if roles and roles.length > 0 %}
-<table>
-    <tr>
-        <th>Role Name</th>
-        <th>Location</th>
-        <th>Capability</th>
-        <th>Band</th>
-        <th>Closing Date</th>
-        <th>Role Status</th>
+<table id="jobRolesTable">
+    <tr id="jobRolesTableHeaders">
+        <th class="headerCol">Role Name</th>
+        <th class="headerCol">Location</th>
+        <th class="headerCol">Capability</th>
+        <th class="headerCol">Band</th>
+        <th class="headerCol">Closing Date</th>
+        <th class="headerCol">Role Status</th>
     </tr>
     {% for role in roles %}
-    <tr>
-        <td>{{ role.roleName }}</td>
-        <td>{{ role.location }}</td>
-        <td>{{ role.capability }}</td>
-        <td>{{ role.band }}</td>
-        <td>{{ role.closingDate }}</td>
-        <td>
+    <tr class="dataRow">
```

Comment:
adding empty css classes just to satisfy tests seems like a bad idea, let's rethink this

Reply:
Refactored test and removed empty css classes